### PR TITLE
56 create volunteer account

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to
#56 #58 #57 
---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/pages/Login/index.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+  useEffect(() => {
+    const getOrganisations = async () => {
+      const { data, error } = await Organisations.getOrganisations();
+
+      if (error) {
+        setState({ httpError: error });
+      } else {
+        setState({
+          organisationsOptions: data.map(({ id, name }) => ({
```
Comment (ashatat) [inline client/src/pages/Login/index.js:50]: should be removed

File: client/src/pages/Signup/Volunteer.js [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+  if (!isDeepEqual(orgsIdsRef.current, organisationsIds)) {
+    orgsIdsRef.current = organisationsIds;
+  }
+
+  useEffect(() => {
+    const getOrganisations = async () => {
+      const { data, error } = await Organisations.getOrganisations();
+
+      if (error) {
+        setState({ httpError: error });
```
Comment (ashatat) [inline client/src/pages/Signup/Volunteer.js:67]:
```suggestion
        setState({ httpError: error.message });
```

File: client/src/theme/global-style.js [hunk 3]
```diff
@@ -24,15 +24,21 @@ const style = css`
...
+  .sb-show-main.sb-main-padded {
+    margin: 0;
+    padding: 0;
+  }
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
```
Comment (ashatat) [inline client/src/theme/global-style.js:39]: dulicated

File: client/src/App.js [hunk 4]
```diff
@@ -1,40 +1,54 @@
...
+                allowedRoles={[roles.VOLUNTEER, roles.HQ]}
+                Component={() => <Pages.Example onLogin={() => {}} />}
+                layout="notification"
+                headerIcon="email"
+              />
+              {/* Auth Routes */}
+              <Route
+                exact
+                path={navRoutes.VOLUNTEER.SIGNUP}
+                Component={() => <Pages.VolunteerSignup />}
```
Comment (ashatat) [inline client/src/App.js:34]:
```suggestion
                Component={Pages.VolunteerSignup}
```

**Review by ashatat — CHANGES_REQUESTED** [review]

File: client/src/theme/global-style.js [hunk 5]
```diff
@@ -24,15 +24,20 @@ const style = css`
...
+    padding: 0;
+  }
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    margin: 0;
   }
```
[the remark this answers is not in the document]
Comment (RamyAlshurafa) [reply client/src/theme/global-style.js]: done

File: client/src/pages/Login/index.js (hunk 1 above)
Comment (RamyAlshurafa) [reply client/src/pages/Login/index.js:50]: done

File: client/src/pages/Signup/Volunteer.js (hunk 2 above)
Comment (RamyAlshurafa) [reply client/src/pages/Signup/Volunteer.js:67]: done

File: client/src/theme/global-style.js (hunk 3 above)
Comment (RamyAlshurafa) [reply client/src/theme/global-style.js:39]: done

File: client/src/App.js (hunk 4 above)
Comment (RamyAlshurafa) [reply client/src/App.js]: done

**Review by ashatat — APPROVED** [review]

**Pull request merged**